### ISSUE TITLE
Detect urls in $ref property in parameters.

### ISSUE THREAD

**Issue by jorgeluis1911** [microcks/microcks#984]
### Describe the bug

I have a yaml with url parameters defined with $ref and a url to the object. Simple example:
![image](https://github.com/microcks/microcks/assets/7100674/87e09ca4-172e-462b-b4f2-0d7e9674de72)

When the importer job import this yaml, throw an error in class OpenAPIImporter and do not process file.
![image](https://github.com/microcks/microcks/assets/7100674/e2b40cff-b614-46ce-bf04-3554c5c05374)

In the $ref it is possible to add 3 types of references and external url is one of them.
https://swagger.io/docs/specification/using-ref/

### Expected behavior

The process does not throw an error, process the file and create the service.

### Actual behavior

The process throw an error and do not process the file.

### How to Reproduce?

Try process any openapi with external url as parameter $ref.

### Microcks version or git rev

1.8.0

### Install method (`docker-compose`, `helm chart`, `operator`, `docker-desktop extension`,...)

helm

**Comment by lbroudoux** [comment]
Thanks for opening this one. We don't fetch absolute external links at the moment. Will see to plan this as an enhancement.

**Comment by jorgeluis1911** [comment]
Hi @lbroudoux, yes i am interested. Thank you very much for your attention.

I have created an example for your testing and attached them to the message. 
[absolute-ref-example.zip](https://github.com/microcks/microcks/files/13346225/absolute-ref-example.zip)

In the file "weather-forecast-openapi-absolute-ref.yaml" I added the line "$ref" into parameters with the URL to the new file "weather-forecast-common.yaml" (it should be uploaded to the "util/openapi" folder) which has the parameter "region".

Regards.
